### PR TITLE
qa_crowbarsetup: Set os-distro property for Magnum image

### PR DESCRIPTION
**Pull request by toabctl**
This must be set according to the documentation:

http://docs.openstack.org/developer/magnum/userguide.html#overview

### CODE REVIEW

**Comment by jgrassler** [comment]
+1

**Comment by toabctl** [comment]
@aspiers can you disable the codeclimate check for ```TODO``` ? Imo adding ```TODO``` **is** useful

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3481,9 +3481,11 @@ function oncontroller_magnum_service_setup ()
 
         . ~/.openrc
 
+        # TODO(toabctl): when replacing the Fedora image, also replace the
+        # os-distro proerty
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: property

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -3481,9 +3481,11 @@ function oncontroller_magnum_service_setup ()
...
         openstack image create --file $service_image_name \
             --disk-format qcow2 --container-format bare --public \
-            magnum-service-image
+            --property os_distro=fedora-atomic magnum-service-image
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: if I read correctly the property expected is 'os-distro'

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: done

File: scripts/qa_crowbarsetup.sh (hunk 2 above)
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: No. It's ```os_distro```

Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: Ah ok, so the documentation is wrong

**Comment by aplanas** [comment]
LGTM but not tested: +1

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: where? still there..

**Comment by dirkmueller** [comment]
+1

**Pull request merged**